buildenv/jenkins/testJobTemplate

```diff
@@ -0,0 +1,164 @@
+#!groovy
+
+/*
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+/**
+ * A template that defines a test job.
+ */
+
+if (!binding.hasVariable('JDK_IMPL')) JDK_IMPL = "openj9"
+if (!binding.hasVariable('ARTIFACTORY_SERVER')) ARTIFACTORY_SERVER = ""
+
+if (binding.hasVariable('LEVELS')) {
+	LEVELS = LEVELS.split(',')
+} else {
+	LEVELS = ['sanity','extended','special']
+}
+
+if (binding.hasVariable('JDK_VERSIONS')) {
+	JDK_VERSIONS = JDK_VERSIONS.split(',')
+} else {
+	JDK_VERSIONS = [8,11,12]
+}
+
+if (binding.hasVariable('GROUPS')) {
+	GROUPS = GROUPS.split(',')
+} else {
+	GROUPS = ['functional','system']
+}
+
+
+if (binding.hasVariable('SPECS')) {
+	SPECS = SPECS.split(',')
+} else {
+	SPECS = ['aix_ppc-64_cmprssptrs','linux_ppc-64_cmprssptrs_le','linux_390-64_cmprssptrs','zos_390-64_cmprssptrs','linux_x86-64_cmprssptrs','linux_x86-64','osx_x86-64_cmprssptrs','win_x86-64_cmprssptrs', 'win_x86']
+}
+
+
+println "LEVELS: ${LEVELS}"
+println "JDK_VERSIONS: ${JDK_VERSIONS}"
+println "GROUPS: ${GROUPS}"
+println "SPECS: ${SPECS}"
+
+def SPEC_JENKINS_FILE_MAP = [ 
+	'aix_ppc-64_cmprssptrs' : 'openjdk_ppc64_aix',
+	'linux_ppc-64_cmprssptrs_le' : 'openjdk_ppc64le_linux',
+	'linux_390-64_cmprssptrs' : 'openjdk_s390x_linux',
+	'zos_390-64_cmprssptrs' : 'openjdk_s390x_zos',
+	'linux_x86-64_cmprssptrs' : 'openjdk_x86-64_linux',
+	'linux_x86-64' : 'openjdk_x86-64_linux_largeHeap',
+	'osx_x86-64_cmprssptrs' : 'openjdk_x86-64_osx',
```

Comment:
There is also an osx_x86-64_cmprssptrs_largeHeap build I believe

Reply:
Also a sparc solaris build coming very shortly (openjdk_sparc_solaris), but I guess we can update this as part of that PR.

Reply:
> There is also an osx_x86-64_cmprssptrs_largeHeap build I believe

Yes, you are right. This map was created and tested in openj9 and openj9 does not build osx_x86-64_largeHeap. I will add it.